screens/party/components/moves.lua

```diff
@@ -74,18 +75,18 @@ local function bind_buttons(nodes, name)
 		local pp = _pokemon.decrease_move_pp(current_pokemon, name)
 		if pp ~= nil then
 			_pokemon.set_move_pp(current_pokemon, name, pp)
-			storage.save()
 			update_pp(current_pokemon, name)
-		end			
+			broadcast.send(messages.SAVE_POKEMON)
```

Comment:
So I am seeing this pattern of broadcasting the message a bunch, but that seems disconnected to me. These modules all know about storage, and they know they got the pokemon FROM storage, so shouldn't they be telling storage that a pokemon has changed?

Seems like it should go:
```
local pkmn = storage.get()
pokemon.do_something(pkmn)
storage.save_pokemon()
```
and then storage shoudl be doing the broadcasting.

Reply:
Could do it like that. I was thinking the profile could take care of loading/saving the data (for `trainer`, `storage` and it self), as it is kind of a "child" to it?